lfucg-exactions/client/js/reducers/accountLedgersReducer.js

```diff
@@ -0,0 +1,38 @@
+import {
+    GET_ACCOUNT_LEDGERS,
+    GET_ACCOUNT_LEDGER_ID,
+    GET_ACCOUNT_LEDGER_QUERY,
+    GET_LOT_ACCOUNT_LEDGERS,
+    GET_ACCOUNT_ACCOUNT_LEDGERS,
+    GET_AGREEMENT_ACCOUNT_LEDGERS,
+    POST_ACCOUNT_LEDGER,
+    PUT_ACCOUNT_LEDGER,
+} from '../constants/apiConstants';
+
+
+const accountLedgersReducer = (state = {}, action) => {
+    const {
+        endpoint,
+    } = action;
+    switch (endpoint) {
+    case GET_ACCOUNT_LEDGER_ID:
+        return action.response;
+    case GET_ACCOUNT_LEDGERS:
+        return action.response;
+    case GET_ACCOUNT_LEDGER_QUERY:
+        return action.response;
+    case GET_LOT_ACCOUNT_LEDGERS:
+        return action.response;
+    case GET_ACCOUNT_ACCOUNT_LEDGERS:
+        return action.response;
+    case GET_AGREEMENT_ACCOUNT_LEDGERS:
+        return action.response;
+    case POST_ACCOUNT_LEDGER:
+        return {};
+    case PUT_ACCOUNT_LEDGER:
+        return {};
+    default:
+        return state;
+    }
```

Comment:
Just a tip to make this a little quicker to code: you can combine case statements (in most languages) if the desired outcome is the same: https://stackoverflow.com/questions/14221778/how-can-i-combine-multiple-cases